.github/dependabot.yml

```diff
@@ -1,5 +1,10 @@
 version: 2
 updates:
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
```

Comment:
[no change needed]

I expect this will create PRs to bump GH action deps sooner than we might get them from upstream Comet BFT. That seems fine to me but I can also see an argument to delay those bumps until we pull from upstream.

Reply:
this is why I think we actually disabled dependabot originally, so I'm not sure we want to renenable it. I think we should if our plan is to deviate from upstream more, but if we aren't going to do that then I think we should leave it off

Reply:
Agreed. IMO we should try to minimize the deviation from upstream. Since bumping GH action deps aren't usually an urgent action, I expect we can wait to pull from upstream most of the time.